src/main/java/pl/noip/lolstats/lol/stats/utils/NoBearerException.java

```diff
@@ -1,4 +1,4 @@
 package pl.noip.lolstats.lol.stats.utils;
 
-public class NoBearerException extends Exception {
+public class NoBearerException extends RuntimeException {
```

Comment:
NoBearerException is not the best name, rename to InvalidAuthrorizationHeaderExpcetion or InvalidBearerHeaderException